test/functional/wallet_listsinceblock.py

```diff
@@ -276,7 +278,8 @@ def test_double_send(self):
         self.sync_all()
 
         # gettransaction should work for txid1
-        self.nodes[0].gettransaction(txid1)
+        tx1 = self.nodes[0].gettransaction(txid1)
+        tx1['blockheight'] = self.nodes[0].getblockheader(tx1['blockhash'])
```

Comment:
As above, the access to the field `"height"` of the RPCs result is missing here.

Reply:
@ryanofsky we should pack our bags 🍻 fixed.